docs/proposals/20220210-md-node-label-sync.md

```diff
@@ -0,0 +1,164 @@
+---
+
+title: Label Sync Between MachineDeployment and underlying Kubernetes Nodes
+authors:
+  - "@arvinderpal"
+  - "@enxebre"
+reviewers:
+  - "@fabriziopandini"
+  - "@neolit123"
+  - "@sbueringer"
+  - "@vincepri"
+  - "@randomvariable"
+creation-date: 2022-02-10
+last-updated: 2022-02-26
+status: implementable
+replaces:
+superseded-by:
+
+---
+
+# Label Sync Between MachineDeployment and underlying Kubernetes Nodes
+
+## Glossary
+
+Refer to the [Cluster API Book Glossary](https://cluster-api.sigs.k8s.io/reference/glossary.html).
+
+## Summary
+
+Labels on worker nodes are commonly used for node role assignment and workload placement. This document discusses a means by which labels placed on MachineDeployments can be kept in sync with labels on the underlying Kubernetes Nodes. 
+
+## Motivation
+
+Kubelet supports self-labeling of nodes via the `--node-labels` flag. CAPI users can specify these labels via `kubeletExtraArgs.node-labels` in the KubeadmConfigTemplate. There are a few shortcomings: 
```

Comment:
>Security concerns have restricted the allowed...

can you add a link to where this is stated?

Reply:
Will do!